dash/development/_r_components_generation.py

```diff
@@ -444,6 +444,9 @@ def write_js_metadata(pkg_data, project_shortname, has_wildcards):
     # create this directory if it is missing
     if not os.path.exists("inst/deps"):
         os.makedirs("inst/deps")
+    else:
+        shutil.rmtree("inst/deps")
+        os.makedirs("inst/deps")
```

Comment:
🌴 
```py
if os.path.exists("inst/deps"):
    shutil.rmtree("inst/deps")

os.makedirs("inst/deps")
```

Reply:
🐠

Reply:
in case you need a more robust tool, try and catch

Reply:
fixed in c8dd6e9